apps/explorer/src/components/transaction-card/TxCardUtils.tsx

```diff
@@ -90,41 +96,34 @@ export function TxAddresses({ content }: { content: LinkObj[] }) {
     );
 }
 
-// Generate table data from the transaction data
-export const genTableDataFromTxData = (results: TxnData[]) => ({
-    data: results.map((txn) => ({
-        date: <TxTimeType timestamp={txn.timestamp_ms} />,
-        transactionId: (
+function TransactionIdContent({ TxnData }: { TxnData: TxnData }) {
+    return (
+        <div className="flex h-full items-center rounded hover:bg-sui-light">
             <TxAddresses
```

Comment:
Rewrite this to be a `TransactionLink` component, and remove the `TxAddresses` component entirely (as it was just used for the `foo -> bar` display.